Lib/cu2qu/test/cu2qu_test.py

```diff
@@ -67,17 +67,17 @@ def test_results_unchanged(self):
         """
 
         expected = {
-            3: 6,
-            4: 26,
-            5: 82,
-            6: 232,
-            7: 360,
-            8: 266,
-            9: 28}
+            2: 6,
+            3: 26,
+            4: 82,
+            5: 232,
+            6: 360,
+            7: 266,
+            8: 28}
 
         results = collections.defaultdict(int)
```

Comment:
On a tangent: looks like collections.Counter is also equivalent of collections.defaultdict(int)